actors/miner/src/lib.rs

```diff
@@ -4251,6 +4342,19 @@ fn validate_precommits(
         )
     })?;
     for (i, precommit) in precommits.iter().enumerate() {
+        // We need to record failures and continue validation rather than continuing the loop in order to:
+        // 1. compute svi
```

Comment:
Is this only for when doing aggregates?

Reply:
Yes, stemming from aggregate prove commit needing the witnesses for sectors that we won't confirm in order to not fail valid sectors